src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java

```diff
@@ -50,13 +51,20 @@
 public class CaseData {
 
 
-    @JsonUnwrapped(prefix = "orderContent")
```

Comment:
Seems like you deleted the prefix for field defined in line 65

Reply:
all fixed